src/main/java/solvd/laba/ermakovich/hf/kafka/producer/KafkaProducer.java

```diff
@@ -0,0 +1,32 @@
+package solvd.laba.ermakovich.hf.kafka.producer;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import reactor.core.publisher.Mono;
+import reactor.kafka.sender.KafkaSender;
+import reactor.kafka.sender.SenderRecord;
+import solvd.laba.ermakovich.hf.event.EventRoot;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+@RequiredArgsConstructor
+@Component
+public class KafkaProducer {
```

Comment:
@kermakovich class is not interfaced

so, you will inject class instead of interface

coupling and hardcoded dependencies is the problem, can lead to disaster in your testing and maintaining process